test/Registry_metadata_test1.js

```diff
@@ -16,11 +16,15 @@ contract('Registry', function(accounts) {
         { 
             let orgName     = "TestName" 
             let serviceName = "ServiceName"
-            let metadata    = "LONG \"BINARY\" STRING 42424242424242424242424242424242424242424242424242424242424"
+            let metadataURI = "LONG \"BINARY\" STRING 42424242424242424242424242424242424242424242424242424242424"
```

Comment:
Is it possible to take the real value for metadataURI from existing ipfs hash has as it varies based on the size/content of the file. This way the test will be similar to the real use case scenario.